packages/cli/lib/services/__snapshots__/model.service.unit.test.ts.snap

```diff
@@ -275,15 +275,15 @@ export interface NangoProps {
     track_deletes?: boolean;
     attributes?: object | undefined;
     logMessages?:
-        | {
```

Comment:
I'm not totally sure why these keep re-formatting. It has happened to me a few times lately.